### PR TITLE
Creator class Update

### PR DESCRIPTION
**Pull request by jaw772**
Updated Creator class to include:
- Game object creator method and tests(Jackson)
- Season object creator method and tests(Jackson)
- Team object creator method and tests(Josh)

### CODE REVIEW

**Review by jacksonvaldez — APPROVED** [review]

File: lib/creator.rb [hunk 1]
```diff
@@ -1,23 +1,65 @@
 require './tg_stat'
+require './game'
+require './team'
+
 class Creator
-  attr_reader :league
```
Comment (tjroeder) [inline lib/creator.rb:3]: No league needed, when we have the teams_hash

**Review by tjroeder — APPROVED** [review]
Good job with the reduction on the #teams_obj_creator.

File: lib/creator.rb [hunk 2]
```diff
@@ -1,23 +1,65 @@
...
+    @seasons_hash = seasons_hash
+    @stats_hash = stats_hash
+    @games_hash = games_hash
   end
 
   def self.create_objects(game_data, team_data, game_team_data)
-    # stat_objects = stat_obj_creator(game_team_data)
-    league = "this is league"
-    self.new(league)
+    stats_hash = self.stat_obj_creator(game_team_data)
```
Comment (jacksonvaldez) [inline lib/creator.rb:16]: very concise. Very clear what the Creator class is doing

**Review by jacksonvaldez — APPROVED** [review]

**Review by Henchworm — COMMENTED** [review]
Nice work gang

**Comment by Henchworm** [comment]
Great work on the final method "Team object creator method and tests"!

**Pull request merged**